devolving_music/urls.py

```diff
@@ -11,6 +11,7 @@
 from .views.unreviewed_duplication_flags import UnreviewedDuplicationFlagView
 
 
+
```

Comment:
Why'd you make this whitespace change? Adds noise to the diff and I think violates PEP 8.